src/wasm-builder.h

```diff
@@ -241,14 +241,16 @@ class Builder {
     auto* ret = allocator.alloc<LocalSet>();
     ret->index = index;
     ret->value = value;
+    ret->makeSet();
     ret->finalize();
     return ret;
   }
-  LocalSet* makeLocalTee(Index index, Expression* value) {
+  LocalSet* makeLocalTee(Index index, Expression* value, Type type) {
     auto* ret = allocator.alloc<LocalSet>();
     ret->index = index;
     ret->value = value;
-    ret->setTee(true);
+    ret->makeTee(type);
+    ret->finalize();
```

Comment:
This `ret->finalize()` is redundant because `makeTee` also calls finalize. Same with `makeSet` above.

Reply:
Done